### ISSUE TITLE
Using hideMissingTitles options breaks the keyboard navigation

### ISSUE THREAD

**Issue by denehyg** [denehyg/reveal.js-menu#7]
As mentioned in issue #6, the addition of the hideMissingTitles option breaks the keyboard navigation - it tries to select the menu item that is missing and fails so it doesn't progress up/down as expected.

**Comment by denehyg** [comment]
Fixed.
